AppConnectSwift/OnePageFormViewController.swift

```diff
@@ -34,7 +34,7 @@ class OnePageFormViewController: UIViewController {
         // NumericFields. If you don't know in advance what the fields are going
         // to be, look at MultiPageFormViewController instead.
         for field in form.fields {
-            switch field.fieldOID {
+            switch (field as AnyObject).fieldOID {
```

Comment:
This compiles? I thought since the type is cast to AnyObject, that `.fieldOID` wouldn't be recognized…